src/Stratis.Bitcoin.Features.MemoryPool/MempoolOrphans.cs

```diff
@@ -180,6 +183,7 @@ public class OrphanTx
         /// <param name="tx">The new transaction received.</param>
         public async Task ProcessesOrphansAsync(MempoolBehavior behavior, Transaction tx)
         {
+            this.mempoolLogger.LogTrace("{0}:'{1}', {2}:'{3}'", nameof(behavior), behavior?.AttachedPeer?.RemoteSocketEndpoint, nameof(tx), tx?.GetHash());
```

Comment:
Attacked peer and behavior can't be null here

Reply:
I think previously AttachedPeer could be null, not sure this is not the case still.

Reply:
It cannot be null, the network peer refactoring that happened a while ago guarantees that all callbacks are finished before detaching.

Reply:
ok yes so it cannot be null remove the `.?`